mne/io/cnt/cnt.py

```diff
@@ -46,14 +46,70 @@ def _read_annotations_cnt(fname, data_format='int16'):
     SETUP_NCHANNELS_OFFSET = 370
     SETUP_RATE_OFFSET = 376
 
+    def _accept_reject_function(keypad_accept):
+        accept_list = []
+        for i in range(len(keypad_accept)):
+            if 'xd0' in str(keypad_accept[i]):
+                accept_list.append('good')
+            elif 'xc0' in str(keypad_accept[i]):
+                accept_list.append('bad')
+            else:
+                accept_list.append('NA')
+        return np.array(accept_list)
+
     def _translating_function(offset, n_channels, event_type,
                               data_format=data_format):
+            
         n_bytes = 2 if data_format == 'int16' else 4
         if event_type == CNTEventType3:
             offset *= n_bytes * n_channels
         event_time = offset - 900 - (75 * n_channels)
         event_time //= n_channels * n_bytes
-        return event_time - 1
+        event_time = event_time - 1
+        #Prevent negative event times
+        for i in range(len(event_time)):
+            if event_time[i] < 0:
+                event_time[i] = 0
+        return event_time
+
+    def _update_bad_span_onset(accept_reject, onset, duration, description):
+        accept_reject = accept_reject.tolist()
+        onset = onset.tolist()
+        duration = duration.tolist()
+        description = description.tolist()
+        # If there are no bad spans, return original onset, duration, description
+        if 'bad' not in accept_reject:
+            return np.array(onset), np.array(duration), np.array(description)
+        #print description
+        print(f'description: {description}')
```

Comment:
if you intend this to be visible to the user, it should be passed to our logger rather than printed.  try `git grep "logger.info"` to see examples of how it's done elsewhere in the codebase. (I'm assuming these should be informational messages, not warnings or errors)

(If you don't want the user to see it and it's just for your use during development/debugging, then it should be removed before this PR is finished)